### PR TITLE
Multi-user support in reference backend as configured in u.p

### PR DESCRIPTION
**Pull request by apurvabisht97**

### CODE REVIEW

**Comment by apurvabisht97** [comment]
**Test cases:-**

1. now multiple user can login to the app simultaneously .

File: v1.0/backend/src/models/users.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            if (value.toLowerCase().includes('password')) {
+                throw new Error('Password cannot contain "password"')
+            }
+        }
+    },
+}, {
+    timestamps: true
+})
+
+userSchema.index({userId: -1})
```
Comment (dileep-gadiraju) [inline v1.0/backend/src/models/users.js:37]: @apurvabisht97 , any specific reason for index({userId: -1}) ? i.e. descending order

File: specs/v1.5/swagger-saral-apidoc.yaml [hunk 2]
```diff
@@ -720,6 +720,10 @@ definitions:
             type: boolean
             description: "If this is set to true , app can operate in offline mode after frist navigation ."
             example: true
+          userId:
```
Comment (dileep-gadiraju) [inline specs/v1.5/swagger-saral-apidoc.yaml:723]: @apurvabisht97 , why did you add userId again?

File: specs/v1.5/swagger-saral-frontend.yaml [hunk 3]
```diff
@@ -227,6 +227,10 @@ definitions:
           district:
             type: string
             example: "district 1"
+          userId:
```
Comment (dileep-gadiraju) [inline specs/v1.5/swagger-saral-frontend.yaml:230]: @apurvabisht97 , why did you add userId again?
